smswebapp/settings/docker.py

```diff
@@ -11,6 +11,11 @@
 # variable is set and is non-empty.
 DEBUG = os.environ.get('SMS_ENABLE_DEBUG_THIS_IS_DANGEROUS', '') != ''
 
+# Redirect all traffic except healthz endpoint to HTTPS unless HTTPS_REDIRECT is set to False
+SECURE_SSL_REDIRECT = os.environ.get('HTTPS_REDIRECT', True) is True
```

Comment:
1. Add this environment variable to the README alongside the others.

2. I think this should be something like:
```python
SECURE_SSL_REDIRECT = os.environ.get('DANGEROUS_DISABLE_HTTPS_REDIRECT') is not None
```
It makes a bit more sense that setting ``DANGEROUS_DISABLE_HTTPS_REDIRECT`` to any value will disable HTTPS redirect but not setting the value will leave it enabled. At the moment setting ``HTTPS_REDIRECT=1`` will, counter-intuitively, disable HTTPS redirect.